### DISCUSSION TITLE
cant upload any stuff in my repository ore anywhere since 3 months even not an image

### DISCUSSION THREAD

**Discussion by kalle07** [community/community#121792] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

4 months ago all working fine
 I was able to i  upload python 
i was abkle to upload an image in a discussion 
now it works nothing

where can i look ?
where i can get support ?

**Comment by ebndev** [comment]
Hi @kalle07,

Do you mean that you can't upload anything onto any website or only GitHub repositories and Discussions? Just wanted to clarify if it's a GitHub specific issue and not your computer!

**Reply by kalle07** [reply]
only on github ...
not drag and drop and not attach file ...
